projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1143,11 +1124,139 @@ function grunion_feedback_admin_notice() {
  */
 class Grunion_Admin {
 	/**
+	 * Define nonce field name
+	 *
+	 * @var string The nonce field name.
+	 */
+	private $export_nonce_field = 'feedback_export_nonce';
+
+	/**
+	 * Instantiates this singleton class
+	 *
+	 * @return Grunion_Admin The Grunion Admin class instance.
+	 */
+	public static function init() {
+		static $instance = false;
+
+		if ( ! $instance ) {
+			$instance = new Grunion_Admin();
+		}
+
+		return $instance;
+	}
+
+	/**
+	 * Grunion_Admin constructor
+	 */
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+		add_action( 'admin_footer-edit.php', array( $this, 'print_export_modal' ) );
+
+		if ( defined( 'JETPACK_BETA_BLOCKS' ) && JETPACK_BETA_BLOCKS || ( defined( 'IS_WPCOM' ) && IS_WPCOM && is_automattician() ) ) {
+			add_action( 'wp_ajax_grunion_export_to_gdrive', array( $this, 'export_to_gdrive' ) );
+		}
+	}
+
+	/**
+	 * Hook handler for admin_enqueue_scripts hook
+	 */
+	public function admin_enqueue_scripts() {
+		$current_screen = get_current_screen();
+		if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'feedback_page_feedback-export' ), true ) ) {
+			return;
+		}
+		add_thickbox();
+		wp_localize_script( 'grunion-admin', 'exportParameters', array( 'exportError' => esc_js( __( 'There was an error exporting your results', 'jetpack' ) ) ) );
+	}
+
+	/**
+	 * Prints the modal markup with export buttons/content.
+	 */
+	public function print_export_modal() {
+		if ( ! current_user_can( 'export' ) ) {
+			return;
+		}
+
+		// if there aren't any feedbacks, bail out
+		if ( ! (int) wp_count_posts( 'feedback' )->publish ) {
+			return;
+		}
+
+		$current_screen = get_current_screen();
+		if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'feedback_page_feedback-export' ), true ) ) {
+			return;
+		}
+
+		$jetpack_logo = new Logo();
+		?>
+		<div id="feedback-export-modal" style="display: none;">
+			<div class="feedback-export-modal__wrapper">
+				<div class="feedback-export-modal__header">
+					<h1 class="feedback-export-modal__header-title"><?php echo esc_html( __( 'Export your Form Responses', 'jetpack' ) ); ?></h1>
+					<p class="feedback-export-modal__header-subtitle"><?php echo esc_html( __( 'Choose your favorite file format or export destination:', 'jetpack' ) ); ?></p>
```

Comment:
In cases like this, you can combo!

```suggestion
					<h1 class="feedback-export-modal__header-title"><?php esc_html_e( 'Export your Form Responses', 'jetpack' ); ?></h1>
					<p class="feedback-export-modal__header-subtitle"><?php esc_html_e( 'Choose your favorite file format or export destination:', 'jetpack' ); ?></p>
```